src/app/api/auth/[...nextauth]/options.ts

```diff
@@ -0,0 +1,36 @@
+import type { NextAuthOptions } from "next-auth";
+import CredentialsProvider from "next-auth/providers/credentials";
+
+export const options: NextAuthOptions = {
+    providers: [
+        CredentialsProvider({
+            name: "Credentials",
+            credentials: {
+                username: {
+                    label: "Username:",
+                    type: "text",
+                    placeholder: "dev"
+                },
+                password: {
+                    label: "Password:",
+                    type: "text",
```

Comment:
Should be resolved by #21 but the type of password should be password
```suggestion
                    type: "password",
```

Reply:
oops

Reply:
shouldnt matter for now